hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/condition/ThresholdRangeConditionEval.java

```diff
@@ -45,6 +45,9 @@ public ThresholdRangeConditionEval(ThresholdRangeCondition condition, NumericDat
         super(condition.match(data.getValue()), data.getTimestamp());
         this.condition = condition;
         this.value = data.getValue();
+        if (this.condition != null) {
+            this.type = this.condition.getType();
+        }
     }
```

Comment:
Hopefully keeping condition type as a member variable will be worth the trouble.  I guess it depends how much people like the targeted message templates.

Note that ExternalCondition will need this change when HWKALERTS-50 work is merged in.